README.md

```diff
@@ -35,14 +35,15 @@ Flags:
       --config string                          config file (default is $HOME/.flowlogs2metrics)  
   -h, --help                                   help for flowlogs2metrics  
       --log-level string                       Log level: debug, info, warning, error (default "error")  
-      --pipeLine.ingest.collector string       Ingest collector API  
       --pipeline.decode.aws string             aws fields  
       --pipeline.decode.type string            Decode type: aws, json, none (default "none")  
       --pipeline.encode.prom string            Prometheus encode API  
       --pipeline.encode.type string            Encode type: prom, none (default "none")  
       --pipeline.extract.aggregates string     Aggregates (see docs)  
       --pipeline.extract.type string           Extract type: aggregates, none (default "none")  
+      --pipeline.ingest.collector string       Ingest collector API  
       --pipeline.ingest.file.filename string   Ingest filename (file)  
+      --pipeline.ingest.kafka string           kafka parameters: brokers, topic, etc  
```

Comment:
please change to `Ingest Kafka API`

The details are in [docs/api.md](https://github.com/netobserv/flowlogs2metrics/pull/55/files#diff-9eddf4dc51bf6a0125ca7fb094468ad284112270385c41cebce4f8f0a29620ab)

Reply:
done